Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -0,0 +1,146 @@
+# ROS2 Fleet Robot Template
+
+A project template that allows building a multi-robot ROS2-enabled simulation. The template contains a level with a large warehouse scene called `Warehouse.prefab`. 
+
+## Requirements
+
+Due to ROS2 dependency, this project was prepared and tested on Linux operating system. It is recomended to use Ubuntu [22.04](https://www.releases.ubuntu.com/22.04/), however any distribution meeting following requirements may be used.
+
+Refer to the [O3DE System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) documentation to make sure that the system/hardware requirements are met. 
+This project has the following dependencies:
+
+- [O3DE](https://github.com/o3de/o3de)
+- [ROS2 Humble](https://www.ros.org)
+- [ROS2 Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2)
+- [Proteus robot](https://github.com/o3de/o3de-extras/tree/development/Gems/ProteusRobot)
+- [WarehouseAssets](https://github.com/o3de/o3de-extras/tree/development/Gems/WarehouseAssets)
+
+Please make sure that `clang` was installed and configured. For details refer to [this section](https://www.o3de.org/docs/welcome-guide/requirements/#linux) of O3DE documentation.
+
+## Setup Instructions
+
+The following steps will assume the following:
+
+- You have ROS2 humble [installed](https://docs.ros.org/en/humble/Installation.html) and environment is [sourced](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files).
+- The O3DE gems, projects and templates will be placed in the O3DE home folder: `/home/${USER}/O3DE`.
+- You have the O3DE engine [downloaded and built](https://www.o3de.org/docs/welcome-guide/setup/setup-from-github) and it is located in `/home/${USER}/O3DE/Engines/Development`.
```

Comment:
Wouldn't $HOME be better to use in this case.
For example if o3de was run with a user account that wasn't placed underneath the `/home` folder?

Reply:
Chnaged to $HOME.